app/migrations/Version20160309144959.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Migrations;
```

Comment:
Migration files [should have license block](https://github.com/Sylius/Sylius/pull/4493) ;)